pkg/actions/irsa/irsa.go

```diff
@@ -26,14 +26,14 @@ func New(clusterName string, stackManager manager.StackManager, oidcManager *iam
 	}
 }
 
-func doTasks(taskTree *tasks.TaskTree) error {
+func doTasks(taskTree *tasks.TaskTree, verb string) error {
```

Comment:
nit: maybe operation or action instead of verb?

Reply:
Yeah I was thinking about the same, but I saw some other methods were doing the same and using `verb`. Anyways, that's changed to `action Action` now :)